serverlessworkflow/modules/ROOT/pages/cloud/operator/build-and-deploy-workflows.adoc

```diff
@@ -17,37 +26,135 @@ This document describes how to build and deploy your Workflow on a Kubernetes cl
+kind: ConfigMap
+metadata:
+  name: sonataflow-operator-builder-config
+----
+
+[WARNING]
+====
+The excerpt above is just an example. The current version might have a slightly different version. Don't use this example in your installation.
+====
+
+=== Changing resources requirements
 
-== Preparing for the build
+You can create or edit a `SonataFlowPlatform` in the workflow namespace specifying the link:{kubernetes_resource_management}[resources requirements] for the internal builder pods:
+
+.Example of SonataFlowPlatform
+[source,yaml,subs="attributes+"]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowPlatform
+metadata:
+  name: sonataflow-platform
+spec:
+  build:
+    resources:
+      requests:
+        memory: "64Mi"
+        cpu: "250m"
+      limits:
+        memory: "128Mi"
+        cpu: "500m"
+----
+
+Every build process will reuse this configuration and start new instances based on it from now on.
+
+[IMPORTANT]
+====
+Only one `SonataFlowPlatform` is allowed per namespace. Try fetching the one the operator created for you and edit it instead of trying to create another one.
+====
+
+You can fine tune the resources requirements for a particular workflow. Every workflow instance will have a `SonataFlowBuild` instance created with the same name as the workflow. You can edit the `SonataFlowBuild` custom resource and specify the same parameters. For example:
+
+.Example of SonataFlowBuild
+[source,yaml,subs="attributes+"]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowBuild
+metadata:
+  name: my-workflow
+spec:
+    resources:
+        requests:
+        memory: "64Mi"
+        cpu: "250m"
+        limits:
+        memory: "128Mi"
+        cpu: "500m"
+----
+
+This parameters will only apply to new build instances.
+
+[#building-kubernetes]
+== Building on Kubernetes
+
+[TIP]
+====
+You can skip this section if you're running <<building-openshift, on OpenShift>>.
+====
 
-You should follow these steps to create a container that you can deploy as a service on Kubernetes.
+Follow these steps to configure your Kubernetes namespace to build workflow images with the operator.
 
 === Create a Namespace for the building phase
 
-Create a new Namespace that will hold all the resources that the operator will create (Pods, Deployments, Services, Secrets, ConfigMap, and Custom Resources) in this guide.
+Create a new namespace that will hold all the resources that the operator will create (Pods, Deployments, Services, Secrets, ConfigMap, and Custom Resources) in this guide.
 
 .Create a Namespace for the application to build and run in
 [source,bash,subs="attributes+"]
 ----
-kubectl create namespace kogito-workflows
-# set the kogito-workflows namespace to your context
-kubectl config set-context --current --namespace=kogito-workflows
+kubectl create namespace workflows
+# set the workflows namespace to your context
+kubectl config set-context --current --namespace=workflows
```

Comment:
Maybe extract the namespace to the variable as well?

Reply:
Not sure if I follow. This way we avoid adding `-n workflows` in the following code snippets.